dev-docs/examples/jw-selfhost-example.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: player_example
+title: JW Self-Hosted Player Integration
+description: Show a prebid ad in a self-hosted JW Player
+
+top_nav_section: dev_docs
+nav_section: quick-start
+
+hide: true
+beta: true
+
+about:
+- Setting up JW Player to dynamically play ads.
+- Using a self-hosted player.
+- Create a masterVideoTag and pass it to invokeVideoPlayer.
+- Use invokeVideoPlayer to set up and access the JW Player instance.
+- Dynamically insert a preroll ad into JW Player.
+
+player_notes:
+- You must have the correct JW Player license that allows you to play advertising.
+- The different methods of embedding JW Player on your site can be found <a href="https://support.jwplayer.com/customer/portal/articles/1406723-mp4-video-embed">here</a>. 
+- For this example we will be using method 3, a self-hosted player and JW Platform hosted content. To see an example using the cloud-hosted player, click <a href="#">here</a>.
+- No matter what embedding method you choose to use, you must follow the <b>custom embed</b> instructions. You cannot use the single-line embed.
+
+jsfiddle_link: jsfiddle.net/shirleylberry/357yaqgc/embedded/html/
+demo_link: http://video-demo.appnexus.com/pbjs/JWPlayerDemo/jwPlayerPrebid.html
```

Comment:
This is rendering as

http://http//video-demo.appnexus.com/pbjs/JWPlayerDemo/jwPlayerPrebid.html

I think it needs the `http://` part removed